Sources/BraveVPN/BraveVPN.swift

```diff
@@ -265,18 +268,29 @@ public class BraveVPN {
 
     // 1. Existing user, check if credentials and connection are available.
     if GRDVPNHelper.activeConnectionPossible() {
-      // just configure & connect, no need for 'first user' setup
-      helper.configureAndConnectVPN { error, status in
-        if let error = error {
-          logAndStoreError("configureAndConnectVPN: \(error)")
+      
+      // Checking if user has to be migrated to wireguard
+      if !Preferences.VPN.wireGuardProtocolMigrated.value {
```

Comment:
Let's remove the migration code. It's a technical detail we do not have to take care of